src/components/FilterSearch.tsx

```diff
@@ -66,7 +66,7 @@ export interface FilterSearchProps {
  */
 export function FilterSearch({
   searchFields,
-  label = 'Filter',
```

Comment:
Just checking that we want to remove the default? (I would be down to remove it, it's not a label I can imagine people using...)